cmd/agent/app/integrations.go

```diff
@@ -293,9 +307,99 @@ func installTuf(cmd *cobra.Command, args []string) error {
 		"--no-deps",
 	}
 
-	tufArgs = append(tufArgs, args...)
+	// Install the wheel
+	if err := tuf(append(tufArgs, args[0])); err != nil {
+		return err
+	}
 
-	return tuf(tufArgs)
+	// Run pip check to determine if the installed integration is compatible with the base check version
+	pipErr := pipCheck()
+	if pipErr == nil {
+		// Success
+		return nil
```

Comment:
nice to have: should we log a nice "success" (in green) message when the installation is successful?